openjdk8/src/main/docker/setup-env.bash

```diff
@@ -1,18 +1,27 @@
 #!/bin/bash
+
+isTrue() {  
+  if [[ ${1,,*} = "true" ]] ;then 
+    return ${true}
+  else 
+    return ${false}
+  fi 
+}
+
 ALPN_BOOT=
-if [[ -n "$ALPN_ENABLE" ]]; then
+if isTrue "${ALPN_ENABLE:=false}" ; then
   ALPN_BOOT="$( /opt/alpn/format-env-appengine-vm.sh )"
 fi
 
 DBG_AGENT=
-if [[ "$GAE_PARTITION" = "dev" ]]; then
-  if [[ -n "$DBG_ENABLE" ]]; then
+if isTrue "${DBG_ENABLE:=$( if [[ -z ${CDBG_DISABLE} ]] ; then echo true; else echo false ; fi )}" ; then
```

Comment:
Fine as is but do you really think we should also consider CDBG_DISABLE as a fallback to DBG_ENABLE?
From the user perspective I thought we said that we only document and consider  DBG_ENABLE (and then
manipulate the other "internal" variables" based on it, as you already do).